lib/ui/form/form_screen.dart

```diff
@@ -105,6 +123,37 @@ class FormScreenState extends ConsumerState<FormScreen> {
                       question: questions[index - 1],
                       questionIndex: index,
                       questionTotal: questionTotal,
+                      onSelectedIndexedAnswers: (selectedIndexes) {
+                        final ansOption = questions[index - 1]
+                            .answers
+                            .map((element) => element.id)
+                            .toList();
+                        final qid = questions[index - 1].id;
+                        List<SubmitSurveyAnswerModel> answerModels = [];
+                        for (int index in selectedIndexes) {
+                          answerModels.add(SubmitSurveyAnswerModel(
+                            id: ansOption[index],
+                            // some answer doesn't have any text, so use index
+                            answer: index.toString(),
```

Comment:
Hmmm, looking from Postman, it should be `null` instead of sending `index`, no? 🙏